code/controllers/subsystem/non_firing/SSmapping.dm

```diff
@@ -53,8 +53,11 @@ SUBSYSTEM_DEF(mapping)
 	lavaland_theme = new lavaland_theme_type
 	log_startup_progress("We're in the mood for [initial(lavaland_theme.name)] today...") //We load this first. In the event some nerd ever makes a surface map, and we don't have it in lavaland in the event lavaland is disabled.
 
-	cave_theme = pick(BLOCKED_BURROWS, CLASSIC_CAVES, DEADLY_DEEPROCK)
-	log_startup_progress("We feel like [cave_theme] today...")
+	var/caves_theme_type = pick(subtypesof(/datum/caves_theme))
+	ASSERT(caves_theme_type)
+	caves_theme = new caves_theme_type
+	log_startup_progress("We feel like [initial(caves_theme.name)] today...")
```

Comment:
Do we need `initial` if we instance it on the line above?

Reply:
Nope, let me fix that for both cave and planet themes.